trade-finance-manager-api/graphql-query-tests/mutation-update-activity-comment.api-test.js

```diff
@@ -0,0 +1,71 @@
+const { createTestClient } = require('apollo-server-testing');
+const { ApolloServer } = require('apollo-server-express');
+const { applyMiddleware } = require('graphql-middleware');
+const { makeExecutableSchema } = require('graphql-tools');
```

Comment:
`graphql-tools` is deprecated. use `@graphql-tools/schema`
i.e. `const { makeExecutableSchema } = require('@graphql-tools/schema');`